docs/docs/client-only-routes-and-user-authentication.md

```diff
@@ -18,9 +18,9 @@ Gatsby converts components in the `pages` folder into static HTML files for the
 
 ## Handling client-only routes with Gatsby
 
-Gatsby uses [@reach/router](https://reach.tech/router/) under the hood, and it is the recommended approach to create client-only routes.
+Gatsby uses [@reach/router](https://reach.tech/router/) under the hood (so you don't need to install it separately), and it is the recommended approach to create client-only routes.
 
-You first need to set up routes on a page that is built by Gatsby:
+You first need to add routes to a page that _is_ built by Gatsby. Below, we'll use `pages/app.js`:
```

Comment:
I tried to rephrase your suggestion per the [style guide](https://www.gatsbyjs.org/contributing/gatsby-style-guide/#use-you-as-the-pronoun). Also, the first sentence reads well already. 
```suggestion
You first need to set up routes on a page that is built by Gatsby. You can see the routes added to  `src/pages/app.js` directory in the code example below:
```

Reply:
@Ekwuno not sure why GitHub let me edit your comment, but I did so to remove an extra space and an extra word, so I'll commit that.